src/less/nav.less

```diff
@@ -245,3 +236,25 @@
 }
 
 }
+
+global-menu-container {
```

Comment:
@mattnickles thats the Global Menu container, where the list of links to Managment, Public Cloud, etc.. go. It should be hidden at that point (if the menu is not open)